File: app/src/main/java/com/example/unittesting/entity/login/LoginValidator.java
package com.example.unittesting.entity.login;

public class LoginValidator {

    static final String EMPTY = "";
    static final int MIN_PASSWORD_LENGTH = 6;

    public boolean validateLogin(String login) {
        return !login.equals(EMPTY);
    }

    public boolean validatePassword(String password) {
        return password.length() >= MIN_PASSWORD_LENGTH;
    }
}

File: app/src/main/kotlin/com/example/unittesting/login/model/LoginValidator.kt
package com.example.unittesting.login.model

class LoginValidator {

    companion object {
        val EMPTY = ""
        val MIN_PASSWORD_LENGTH = 6
    }

    fun validateLogin(login: String): Boolean {
        return login != EMPTY
    }

    fun validatePassword(password: String): Boolean {
        return password.length >= MIN_PASSWORD_LENGTH
    }
}
